### PR TITLE
Update README.md for better organization

### PR DESCRIPTION
**Pull request by jasonbasuil**
Updating the README for better organization and readability. Also removes some irrelevant info and hopefully should be easier to consume. Some sections are still WIP but hoping to get those filled in by end of the month with some helpful runbooks.

Open to suggestions for edits or things to include. Looking to make this as helpful as possible, especially for newcomers to the DS.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,92 +1,104 @@
...
-Creates symlinks for packages needed to develop/debug against our `rails-server` repository.
+If you want to flag anything design system related on any PR you're working on or have questions/suggestions, you can tag @user-interviews/design-system and we will help out!
 
-### `yarn rs:link:watch`
 
-Trigger a production build when files are changed.
+## Storybook development (Under construction 🚧)
 
-### `yarn rs:unlink`
+<a href='https://docs.userinterviews.com/?path=/story/intro--page'>
```
Comment (brianCollinsUI) [inline README.md:93]: **nitpick**: 💅

Is this the right url?

Screenshot 2024-08-20 at 1 21 07 PM


do we want https://docs.userinterviews.com/?path=/docs/intro--docs for this?

File: README.md [hunk 2]
```diff
@@ -1,92 +1,104 @@
...
+</a>
 
-Remove symlinked packages.
+## Chromatic visual testing (Under construction 🚧)
 
-## Chromatic visual testing & live feedback
 We use [Chromatic](https://www.chromatic.com/) for visual testing and gathering UI feedback.
 
-To join Chromatic for 'ui-design-system', ask Jason for the join link.
+To join Chromatic for 'ui-design-system', ask the design system lead for the join link.
```
Comment (brianCollinsUI) [inline README.md:101]: **praise**: 🙌

**Review by brianCollinsUI — APPROVED** [review]
Nice - very easy to follow this readme

File: README.md [hunk 3]
```diff
@@ -1,92 +1,104 @@
...
-- Push your changes to Github and submit for a review / pull request.
+`yarn rs:link`
+_Creates symlinks for packages needed to develop/debug against our `rails-server` repository._
 
-If you need any help with setup or have questions about the process of writing documentation, feel free to reach out to a developer or a member of the Design System team.
+`yarn rs:link:watch`
+_Trigger a production build when files are changed._
 
+`yarn rs:unlink`
+_Remove symlinked packages when you're finished._
```
Comment (Dchyk) [inline README.md:61]: 💭 it might make sense to add a note about the setup scripts that exist in the repo here, eg `bin/migrate-stack`

**Review by Dchyk — APPROVED** [review]
nice work, this is a big improvement!

**Review by kyleshike — APPROVED** [review]

**Pull request merged**